src/shared/ComboButton/dropdown.jsx

```diff
@@ -7,16 +7,29 @@ export const DropDown = props => {
   return <div className="dropdown">{props.children}</div>;
 };
 
-export const DropDownItem = props => {
-  const liClasses = props => classNames({ disabled: props.disabled });
-  return <p className={liClasses(props)}>{props.value}</p>;
-};
+export class DropDownItem extends Component {
+  handleClick = () => {
+    if (!this.props.disabled) {
+      this.props.onClick();
+    }
+  };
+
+  render() {
+    const liClasses = props => classNames({ disabled: props.disabled });
+    return (
+      <p className={liClasses(this.props)} onClick={this.handleClick}>
```

Comment:
```suggestion
      <p className={liClasses} onClick={this.handleClick}>
```

Reply:
I think this also makes more sense as a div and for the classNames to be called something like `dropdownItemClasses`

Reply:
Instead of having a `handleClick` method to determine whether or not we use `onClick` if its disabled. We could instead decide whether or not we provide an `onClick` prop if it is disabled 
```suggestion
      <p className={liClasses(this.props)} onClick={props.disabled ? props.onClick : null}>
```

Reply:
good call. started as <li>'s :) as you can probably guess. 

sure re: div change. Why does make more sense as a div (just curious)? Was just copying wireframe on structure.

Reply:
Just semantically a div is usually the go to for something mimicking something you can interact with. A p is usually more used for dividing content

Reply:
👍 re: div. 
➕ idea of disabling onClick via the props.